File: PROGRAMMERS/Level 1/제일 작은 수 제거하기/PROGRAMMERS/PROGRAMMERS.cpp
// PROGRAMMERS.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <vector>
#include <iostream>

using namespace std;

int main()
{
	int nLast = -1;
	vector<int> arr = { 4,3,2,1 };
	vector<int> answer;


	int nMin = arr[0];
	int nPos = 0;

	for (int i = 1; i < arr.size(); i++)
	{
		if (nMin > arr[i])
		{
			nMin = arr[i];
			nPos = i;
		}
	}

	if (arr.size() == 1)
	{
		answer.push_back(-1);
	}
	else
	{
		arr.erase(arr.begin() + nPos);
		answer = arr;
	}

    return 0;
}

File: PROGRAMMERS/Level 1/최대공약수와 최소공배수/PROGRAMMERS.py
answer = ''
n = 2
m = 5

nMax = -1
for i in range(1, min(n,m) + 1):
    if(n % i == 0 and m % i == 0):
        if(nMax < i ):
            nMax = i;

nMin = (n * m) / nMax
answer = [nMax, nMin]